[OSS] feat(cli): add initial peering cli commands (#14423)
hashicorp/consul: website/content/docs/connect/cluster-peering/create-manage-peering.mdx

</Tab>

<Tab heading="Consul CLI">

> In one of the client agents in "cluster-02," use recorded token and the [`consul peering establish` command](/commands/peering/establish) to establish the peering connection.

Reviewer: ```suggestion
In one of the client agents in "cluster-02," issue the [`consul peering establish` command](/commands/peering/establish) and specify the token generated in the previous step. The command establishes the peering connection.
```

> The commands will print "Successfully established peering connection with cluster-01" when the connection is established.

Reviewer: ```suggestion
The commands prints "Successfully established peering connection with cluster-01" after the connection is established.
```
